src/integration-test/java/apptest/CheckPermitsIT.java

```diff
@@ -0,0 +1,89 @@
+package apptest;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import se.sundsvall.dept44.test.AbstractAppTest;
+import se.sundsvall.dept44.test.annotation.wiremock.WireMockAppTestSuite;
+import se.sundsvall.precheck.Application;
+
+@WireMockAppTestSuite(
+        files = "classpath:/PrivatePreCheck/",
+        classes = Application.class
+)
+class CheckPermitsIT extends AbstractAppTest {
+
+    private final String VALID_PRE_CHECK_ID = "d1d87cda-0dc5-41bb-9dd9-fa6ed2fd52ee";
+    private final String VALID_MUNICIPALITY_ID = "2281";
+    private final String TEMPLATE_SERVICE_PATH = "/PreCheck/%s?municipalityId=%s&assetType=%s";
+
+    @BeforeEach
+    void setUp() {
+        CommonStubs.stubForAccessToken();
+    }
+
+    @AfterEach
+    void tearDown() {
+        verifyAllStubs();
```

Comment:
This is not necessary.

(Even if it was necessary: verification of stubs should not be performed in a teardown-method)